Sources/ProjectSpec/Breakpoint.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+import XcodeProj
+import JSONUtilities
+
+public struct Breakpoint: Equatable {
+
+    public struct Action: Equatable {
+
+        public var type: XCBreakpointList.BreakpointProxy.BreakpointContent.BreakpointActionProxy.ActionExtensionID
```

Comment:
Maybe for now we can create 2 typealiases that we can use throughout the codebase
```
typealias BreakpointActionType XCBreakpointList.BreakpointProxy.BreakpointContent.BreakpointActionProxy.ActionExtensionID
typealias BreakpointType = XCBreakpointList.BreakpointProxy.BreakpointExtensionID
```